admin/services/pin-generation-v2.service.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
```

Comment:
@jon-shipley same question as above regarding the tests for this service.

Reply:
tests added